Dockerfile

```diff
@@ -3,7 +3,8 @@ FROM praekeltfoundation/django-bootstrap:py3.6
 COPY . /app
 COPY entrypoint.sh /scripts/django-entrypoint.sh
 COPY nginx.conf /etc/nginx/conf.d/django.conf
-RUN pip install -e .
+RUN pip install -r requirements.txt
+RUN pip install -r requirements-dev.txt
```

Comment:
I don't think we want the `dev` dependencies in the Docker container

Reply:
Good catch, thanks!